content/en/synthetics/browser_tests/actions.md

```diff
@@ -114,15 +111,15 @@ Create this assertion step to have your browser test verify that the URL of the
 
 You can test for a value in the URL such as `string`, `number`, or `regex`.
 
-{{% /tab %}}
+{{< /collapse-content >}}
 
-{{% tab "Special Assertions" %}}
+{{% collapse-content title="Special Assertions" level="h4" %}}
```

Comment:
sure! I can remove the collapse feature, search results still work, I thought it would help to clean up the page a little but yes agree content could get missed, thanks for the feedback!